### PR TITLE
Allow to listen current time at video transcoding process

### PR DESCRIPTION
**Pull request by khoben**
Extend composer event listener with onCurrentWrittenVideoTime
May used in timed filters, e.g. video captions

### CODE REVIEW

**Review by MasayukiSuda — APPROVED** [review]
Thanks great pull request !! Please resolve conflict.

**Comment by MasayukiSuda** [comment]
@khoben 
Please resolve conflict.

File: mp4compose/src/main/java/com/daasuu/mp4compose/composer/Mp4ComposerEngine.java [hunk 1]
```diff
@@ -281,8 +281,8 @@ private void runPipelines() {
                 if (progressCallback != null) {
                     progressCallback.onCurrentWrittenVideoTime(writtenPresentationVideoTimeUs);
```
Comment (khoben) [inline mp4compose/src/main/java/com/daasuu/mp4compose/composer/Mp4ComposerEngine.java:282]: I think here we should catch original timestamp, not trimmed value, seems like that.

Comment (MasayukiSuda) [reply mp4compose/src/main/java/com/daasuu/mp4compose/composer/Mp4ComposerEngine.java:282]: Thanks !! please fix this...

**Pull request merged**